release-notes/release-notes56.md

```diff
@@ -87,6 +80,24 @@ For information on customizing autocompleter results with `CardMenuItem`, see: [
 
 Clicking links when the editor is in `readonly` mode will open the URL in a new tab or window (depending on browser preferences).
 
+### Minor new features
+
+{{site.productname}} 5.6 introduces the following minor new features:
+
+- Added support for `block` and `unblock` methods on inline dialogs.
```

Comment:
This should maybe in enhancements? as it's not a new API it's honestly more of a bug fix than anything.

Reply:
Moved in https://github.com/tinymce/tinymce-docs/pull/1781/commits/8d61ea7b793c885b0a66b742cde296a5ef7dbc03